File: remove_covered_intervals/intervals.py
class Solution(object):
    def removeCoveredIntervals(self, intervals):
        """
        :type intervals: List[List[int]]
        :rtype: int
        """
        n = len(intervals)
        result = n
        for i in range(n):
            for j in range(n):
                  if i != j and intervals[i][0] >= intervals[j][0] and intervals[j][1] >= intervals[i][1]:
                        result -= 1
                        break
        return result

File: remove_covered_intervals/intervals.cs
public class Solution {
    public int RemoveCoveredIntervals(int[][] intervals) {
        int n = intervals.Length;
        int result = n;

        for(int i = 0; i < n; i++)
            for(int j = 0; j < n; j++){
                if(i != j && intervals[i][0] >= intervals[j][0]
                && intervals[j][1] >= intervals[i][1]){
                    result -= 1;
                    break;
                }
            }
        return result;
    }
}
